src/test/java/games/strategy/triplea/settings/SaveFunctionTest.java

```diff
@@ -0,0 +1,114 @@
+package games.strategy.triplea.settings;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import javax.swing.JOptionPane;
+
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.core.Is;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import com.google.common.collect.ImmutableMap;
+
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
We've recently switched all Mockito test runners to `MockitoJUnitRunner.StrictStrubs.class`.